packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -322,7 +337,8 @@ class TextSpan extends InlineSpan {
 
   @override
   void describeSemantics(Accumulator offset, List<int> semanticsOffsets, List<dynamic> semanticsElements) {
-    if (recognizer != null && (recognizer is TapGestureRecognizer || recognizer is LongPressGestureRecognizer)) {
+    if (recognizer != null &&
+      (recognizer is TapGestureRecognizer || recognizer is LongPressGestureRecognizer)) {
```

Comment:
same here

Reply:
Where? I do not see it. :)

Reply:
hmmm I might be wrong but I remember it should be something like
```dart
if (
    recognizer != null &&
    (recognizer is TapGestureRecognizer || recognizer is LongPressGestureRecognizer)
)
```